services/GovDeliveryService.ts

```diff
@@ -48,6 +52,53 @@ export interface PublishingSupportEmail {
   apiInternalOnlyDetails?: string;
   apiOtherInfo?: string;
 }
+export interface ContactDetails {
+  firstName: string;
+  lastName: string;
+  email: string;
+}
+export interface ProductionAccessSupportEmail {
```

Comment:
i would love to move these to the `types` directory. don't have to migrate the old ones, but i think we can put the new ones there

Reply:
that's a good idea. I'll move them